src/test/regress/expected/subquery_and_cte.out

```diff
@@ -471,6 +478,524 @@ END;
 $$;
 ERROR:  (3/3) failed to execute one of the tasks
 CONTEXT:  PL/pgSQL function inline_code_block line 29 at RAISE
+--CTEs can be used as a recurring tuple with subqueries in WHERE
+WITH event_id AS (
+	SELECT user_id as events_user_id, time as events_time, event_type
+	FROM events_table
+)
+SELECT
+	count(*)
+FROM
+	event_id
+WHERE
+	events_user_id IN (SELECT user_id FROM users_table);
+ count 
+-------
+   101
+(1 row)
+
+--Correlated subqueries can not be used in WHERE clause
+WITH event_id AS (
+	SELECT user_id as events_user_id, time as events_time, event_type
+	FROM events_table
+)
+SELECT
+	count(*)
+FROM
+	event_id
+WHERE
+	events_user_id IN (SELECT user_id FROM users_table where users_table.time = events_time);
+ERROR:  cannot pushdown the subquery
+DETAIL:  Complex subqueries and CTEs are not allowed in the FROM clause when the query has subqueries in the WHERE clause and it references a column from another query
+-- Recurring tuples as empty join tree
+SELECT
```

Comment:
The formatting of the queries is too wide and don't look like the existing tests in the other files. Could you write slightly denser queries with less white spaces?